flags.go

```diff
@@ -57,7 +57,7 @@ var (
 	}
 	flStrategy = cli.StringFlag{
 		Name:  "strategy",
-		Usage: "placement strategy to use [binpacking, random]",
+		Usage: "PlacementStrategy to use [balanced, binpacking, random]",
```

Comment:
While you add the tests, can you switch this back to `placement strategy` (lowercase + space)